tempesta_fw/http_parser_eol.h

```diff
@@ -0,0 +1,139 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2016 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+__FSM_STATE(RGen_EoL) {
+	parser->_tmp.eol = 0;
+	/* Pass through */
+}
+__FSM_STATE(RGen__EoL) {
+	if (likely(IS_CR_OR_LF(c))) {
```

Comment:
The macro is defined in http_parser.c and also used here and there, so the file is not a separate logic, but just a piece of large logic from http_parser.c. So I agree with @keshonok to move the file content to http_parser.c